src/FeatureFlags/OptionBasedFormEditor/Settings/DefaultOptions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Give\FeatureFlags\OptionBasedFormEditor\Settings;
+
+/**
+ * @unreleased
+ */
+class DefaultOptions extends AbstractOptionBasedFormEditorSettings
+{
+    /**
+     * @unreleased
+     */
+    public function getDisabledOptionIds(): array
+    {
+        return [
+            //Post Types Section
+            'form_featured_img',
+            'featured_image_size',
+            'form_sidebar',
+            //Terms and Conditions Section
+            'terms',
+        ];
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getDisabledSectionIds(): array
+    {
+        return [
+            'display-settings',
```

Comment:
This one turns out we actually do use one setting 😬 :
- `Title Prefixes`: currently used in the donor name block

Reply:
Resolved in 512ad4dc6a0e8c13cfb06b4553ddf969ab81e0fd